api/app/Generators/PoolCandidateCsvGenerator.php

```diff
@@ -269,4 +261,35 @@ private function generatePoolHeaders()
                 }
             });
     }
+
+    private function buildQuery()
+    {
+        $query = PoolCandidate::with([
+            'generalQuestionResponses' => ['generalQuestion'],
+            'screeningQuestionResponses' => ['screeningQuestion'],
+            'educationRequirementExperiences',
+            'user' => [
+                'department',
+                'currentClassification',
+                'userSkills' => ['skill', 'experiences' => ['skills']],
+                'awardExperiences' => ['userSkills' => ['skill']],
+                'communityExperiences' => ['userSkills' => ['skill']],
+                'educationExperiences' => ['userSkills' => ['skill']],
+                'personalExperiences' => ['userSkills' => ['skill']],
+                'workExperiences' => ['userSkills' => ['skill']],
+            ],
+        ]);
+
+        $query = $this->applyFilters($query, PoolCandidate::class, [
+            'priorityWeight' => 'candidateCategory',
+            'poolcandidateStatus' => 'poolcandidateStatuses',
```

Comment:
The casing seems wrong here.

Reply:
Good catch, fixed in [fix casing for status scope map](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11300/commits/0f4cdcd48f6da194ce2e8568a608355c427f8329)